hw/top_earlgrey/data/pins_cw310.xdc

```diff
@@ -43,22 +47,33 @@ set_property -dict { PACKAGE_PIN W9 IOSTANDARD LVCMOS18 }  [get_ports { IOB2 }];
 #set_property -dict { PACKAGE_PIN W10 IOSTANDARD LVCMOS18 } [get_ports {  }]; #USRDIP6
 #set_property -dict { PACKAGE_PIN V11 IOSTANDARD LVCMOS18 } [get_ports {  }]; #USRDIP7
 
-## SPI / JTAG (part of it, other JTAG signals further below)
+## SPI
 set_property -dict { PACKAGE_PIN D26 IOSTANDARD LVCMOS33 } [get_ports { SPI_DEV_CLK }]; #SCK (SPI1_SCK)
 set_property -dict { PACKAGE_PIN A24 IOSTANDARD LVCMOS33 } [get_ports { SPI_DEV_D0 }]; #SDI (SPI1_COPI)
 set_property -dict { PACKAGE_PIN A22 IOSTANDARD LVCMOS33 } [get_ports { SPI_DEV_D1 }]; #SDO (SPI1_CIPO)
 set_property -dict { PACKAGE_PIN C26 IOSTANDARD LVCMOS33 } [get_ports { SPI_DEV_CS_L }]; #CSB (SPI1_CS)
 
-# JTAG (second part)
-set_property -dict { PACKAGE_PIN  U24  IOSTANDARD LVCMOS33 PULLTYPE PULLUP } [get_ports { IOR4 }];       #USB_A13 (SAM3X)
-set_property -dict { PACKAGE_PIN  U22  IOSTANDARD LVCMOS33 PULLTYPE PULLUP } [get_ports { IO_JSRST_N }]; #USB_A14 (SAM3X)
-# SW Straps
+## JTAG
+set_property -dict { PACKAGE_PIN  T17  IOSTANDARD LVCMOS33 PULLTYPE PULLUP } [get_ports { JTAG_SRST_N }]; #USR_DBG_nRST (nSRST)
+set_property -dict { PACKAGE_PIN  U19  IOSTANDARD LVCMOS33 PULLTYPE PULLUP } [get_ports { IOR4 }];       #USR_DBG_TDAT1 (nTRST)
+set_property -dict { PACKAGE_PIN  N18  IOSTANDARD LVCMOS33 PULLTYPE PULLUP } [get_ports { IOR3 }];       #USR_DBG_TCK
+set_property -dict { PACKAGE_PIN  R16  IOSTANDARD LVCMOS33 PULLTYPE PULLUP } [get_ports { IOR2 }];       #USR_DBG_TDI
+set_property -dict { PACKAGE_PIN  P16  IOSTANDARD LVCMOS33 PULLTYPE PULLUP } [get_ports { IOR1 }];       #USR_DBG_TDO
+set_property -dict { PACKAGE_PIN  N16  IOSTANDARD LVCMOS33 PULLTYPE PULLUP } [get_ports { IOR0 }];       #USR_DBG_TMS
+#set_property -dict { PACKAGE_PIN  U24  IOSTANDARD LVCMOS33 PULLTYPE PULLUP } [get_ports { IOR4 }];       #USB_A13 (SAM3X), TRST_N
```

Comment:
we may want to stick in a `// TODO(<issue nr>)` comment here, linking to the issue describing the JTAG on SAM3X problem, and that we should address this in the future...

Reply:
Done.